specs/subresourceintegrity/spec.markdown

```diff
@@ -332,8 +332,8 @@ One should note that being a [secure document][] (e.g. a document delivered
 over HTTPS) is not necessary for the use of integrity validation. Because
 resource integrity is only an application level security tool, and it does not
 change the security state of the user agent, a [secure document] is
-unnecessary. However, if integrity is used in other than a [secure document][] (e.g.
-a document delivered over HTTP), authors should be aware that the integrity
+unnecessary. However, if integrity is used in somethig else than a [secure document][]
+(e.g. a document delivered over HTTP), authors should be aware that the integrity
```

Comment:
And is there a difference when it's inside parentheses though? This [Oatmeal comic](http://theoatmeal.com/comics/ie) (I know, not the most authoritative source) seems to say there is:

![An example of e.g.](https://s3.amazonaws.com/theoatmeal-img/comics/ie/4.png)